### PR TITLE
View a single book

### PR DESCRIPTION
**Pull request by andyw8**
https://trello.com/c/A196Qdzw/6-view-a-single-book

Allows user to see details about the book which aren't on the index
page.

### CODE REVIEW

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -2,4 +2,9 @@ class BooksController < ApplicationController
   def index
     render locals: { books: Book.all }
   end
+
+  def show
+    book = Book.find(params[:id])
+    render locals: { book: book }
```
Comment (taylorrf) [inline app/controllers/books_controller.rb]: like that "render locals:”, normally I use global vars

**Comment by taylorrf** [comment]
Good job :+1:

File: app/controllers/books_controller.rb (hunk 1 above)
Comment (andyw8) [inline app/controllers/books_controller.rb]: I think you mean instance variables :smile:

Comment (taylorrf) [inline app/controllers/books_controller.rb]: Yes, it's instance variables :joy:

**Pull request merged**